go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	k8s.io/client-go v0.28.3
 	k8s.io/component-base v0.28.3
 	sigs.k8s.io/controller-runtime v0.16.3
-	sigs.k8s.io/gateway-api v1.0.0-rc2
+	sigs.k8s.io/gateway-api v1.0.0-rc2.0.20231027061746-5fd8bec3bedc
```

Comment:
Let's not bump this in a PR not related to Gateway API

Reply:
I checked out to the branch from another branch to update the conformance tests. I will make another PR from main branch and close this.